app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from typing import List
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: List[dict], cafe: Cafe) -> str:
```

Comment:
It's a good practice to use type annotations for function arguments and return values. However, the type of the `friends` argument is not clear enough. It would be better if you specify the type of the elements in the list. For example, if `friends` is a list of `Friend` objects, you should annotate it like this: `List[Friend]`.